test/metadata_test.rb

```diff
@@ -84,5 +84,42 @@ def setup
       assert_equal "Friendly Name", req_attr.attribute("FriendlyName").value
       assert_equal "Attribute Value", REXML::XPath.first(xml_doc, "//md:AttributeValue").text.strip
     end
+
```

Comment:
it might be nice for cleanliness to instantiate the `settings` object in a `let` block, and do the setup for these tests in a `before` block

```ruby
describe "when the settings indicate to sign (embedded) the metadata" do
  let(:settings) { OneLogin::RubySaml::Settings.new }
  
  before do
    settings.issuer = "https://example.com"
    # ... etc
  end

  it "creates a signed metadata" do
    # your tests
  end
```